### ISSUE TITLE
Is it possible to force escaping double square brackets in code blocks?

### ISSUE THREAD

**Issue by marcemv90** [EddyLuten/mkdocs-alias-plugin#1]
Hello 👋 

First of all congratulations for the great plugin, I find it very useful. But now I'm facing something that maybe wasn't taken into account when the plugin was developed, or maybe I'm doing something wrong. I hope somebody can shed some light over this.

The issue happens when I want to put literally `[[ something ]]`, enclosed by double square brackets inside a code block.

Specifically, I want to put a code block with something like:

~~~
```shell
   [[ ! -d $HOME/myfolder ]] && mkdir $HOME/myfolder
```
~~~

which renders just fine in Markdown, but the MKDocs console prints it as a warning because the double square brackets are interpreted as an alias that doesn't exist.

```shell
WARNING  -  Alias ' ! -d $HOME/myfolder ' not found in 'test.md'
```

I know it can be legit using aliases within code blocks, in fact I use them, but just wondering if there could be a way to force escaping them, so they are not interpreted as an alias. As said, this is not a big deal, as the website renders good, it is just to avoid that WARNING message in the console log.

Great job with the plugin! 💪 

Thanks and regards.

**Comment by EddyLuten** [comment]
This is now fixed in v0.4.0 using a new escape syntax. Let me know if this works for you.

**Comment by marcemv90** [comment]
> This is now fixed in v0.4.0 using a new escape syntax. Let me know if this works for you.

Hello @EddyLuten,

I just tried the 0.4.0 release and it seems to work like a charm, no WARNING message in the mkdocs console and the website renders good.

Many thanks for the quick fix!

Regards.
